source/speech/extensions.py

```diff
@@ -7,10 +7,16 @@
 Extension points for speech.
 """
 
-from extensionPoints import Action
+from extensionPoints import Action, Filter
 
 speechCanceled = Action()
 """
 Notifies when speech is canceled.
 Handlers are called without arguments.
 """
+
+filterSpeechSequence = Filter()
```

Comment:
I think you can subscript this now to make type hints work.

```suggestion
filterSpeechSequence = Filter[SpeechSequence]()
```